fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonUtils.xtend

```diff
@@ -415,6 +415,13 @@ class PythonUtils {
                 «ELSE»
                 pass
                 «ENDIF»
+                «IF isDynamic»
+                """   Example how to write dynamic attribute
+                Copy inside the following code to protected area
+                name = w_attr.get_name()
+                self.attr_«attribute.name»_read = w_attr.get_write_value()
```

Comment:
The example attribute name ending with `_read` is a bit confusing for a read/write attribute.  I see that is what the old dynamic example code proposes in `initialize_dynamic_attributes`.  Consider dropping the `_read` suffix from the read, write and initialise examples.

Reply:
I like your proposition, so this will be a good candidate for the next PR. 
My plan is to create a new task to discuss what this suffix should be changed (I don't think this is in the scope of this task).
But if you want we can do this in this current PR. If not, let's move the discussion to the next task that I will create, 

What do you think?

Reply:
Yes, it can wait for another PR.